scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 
     # base64 encode
```

Comment:
Using 'cat' with a pipe is unnecessary here since 'base64' can read files directly. The original version using 'base64 -w 0' is more efficient as it avoids creating an extra process.